en/modals.ftl

```diff
@@ -4,13 +4,15 @@
 
 ## Modal for generating a custom mask
 
+modal-custom-alias-picker-tip = Tip: To create a custom mask anytime, make it up on the spot. If you use your { -brand-name-firefox-relay } domain, it’ll work, even if you didn’t generate it here first.
```

Comment:
@rafeerahman I did a check of the reference to domain, it usually use Relay, not Firefox Relay: https://pontoon.mozilla.org/zh-CN/firefox-relay-website/all-resources/?search=domain&string=243297 - see the search result on the left panel. We probably should change to `{ -brand-name-relay } domain`. The previous comment was an example what Fluent brand format looks like.

Reply:
I see, `{ -brand-name-relay }` is correct. Is it ok to use https://github.com/mozilla-l10n/fx-private-relay-l10n/commit/1ebc8676877ff061e2ef2e5c615332dee8b4893b as a reference for the future?

Reply:
Please refer to this file: https://github.com/mozilla-l10n/fx-private-relay-l10n/blob/main/en/brands.ftl - the section in the file you mentioned should have been removed after the brands.ftl was created. This way, we can stay consistent across all the files to refer to this one only. One change will affect all the others.